packages/core/src/State.ts

```diff
@@ -39,7 +43,9 @@ export function isStateConfig<
 export const isState = isStateConfig;
 export class State<
   TContext extends MachineContext,
-  TEvent extends EventObject = EventObject,
+  TEvent extends EventObject,
+  _TAction extends ParameterizedObject,
```

Comment:
This should be removed from here. I think that `State` won't actually depend on `TAction` (it doesn't right now and some of the recent changes were specifically decoupling `State` from actions).

Reply:
I removed them